lineman/app/components/thread_preview_collection/thread_preview_collection.scss

```diff
@@ -3,3 +3,10 @@
   margin-bottom: $cardPaddingSize;
   margin-top: $thinPaddingSize;
 }
+
+.discussions-card__list .thread-previews-container{
+  @include cardNoPadding;
+  margin: $thinPaddingSize (-1 * $cardPaddingSize);
```

Comment:
Is the negative margin the best thing to do? I've tried to keep negative margins out of the CSS so far.. and it's kind of jangy to run in to them sometimes and change how I think about doing CSS..

Happy to use em if you think it's the best thing to do but I don't want to end up with things breaking when moved between cards.